doc/user_guide/ecosystem.rst

```diff
@@ -59,3 +59,8 @@ Starborn is a Python visualization library based on Vega and Altair that aims to
 .. List of links.
 .. _starborn: https://github.com/PythonCharmers/starborn
 .. _Seaborn: https://seaborn.pydata.org/index.html
+
+gif_
```

Comment:
This link needs a target. You can model it after `starborn` and `Seaborn` above.